pipelines/README.md

```diff
@@ -64,15 +63,54 @@ oc create -f azureml-container-pipeline/azureml-container-pipelinerun-tensorflow
 - Apply build-secret. E.g.:
 
 ```bash
-oc apply -f <downloaddir>/rhoai-edge-build-secret.yml 
+oc apply -f <downloaddir>/rhoai-edge-build-secret.yml
 oc secret link pipeline rhoai-edge-build-pull-secret
 ```
 
 ### Deploy and run the test pipeline
 
-> **NOTE** Make sure to change the `target-imagerepo` parameter to match the name of your Quay namespace if using one of the provided `PipelineRun` files.
+> **NOTE**
+> Make sure to change the `target-imagerepo` parameter to match the name of your Quay namespace if using one of the provided `PipelineRun` files.
+
+```bash
+oc apply -k tekton/test-mlflow-image-pipeline/
+oc create -f tekton/test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml
+```
+
+## Deploy the GitOps pipeline
+
+### Git Repository and Credentials
+
+- Clone/Mirror this repository on your Git server
+- Change the provided `gitops-git-user-` files to match your Git credentials
+
+### Deploy the GitOps pipeline
 
 ```bash
-oc apply -k test-mlflow-image-pipeline/
-oc create -f test-mlflow-image-pipeline/test-mlflow-image-pipelinerun-tensorflow-housing.yaml
+oc apply -k tekton/gitops-update-pipeline/
+```
+
+#### Run the GitOps pipeline
+
+The pipeline can be run through the OpenShift Pipelines web console, by specifying all of the details for your environment.
```

Comment:
Doesn't this apply to all of the pipelines? Why state it explicitly here?

Reply:
I'll remove it